bundles/org.openhab.core.test.magic/src/main/resources/ESH-INF/thing/channel-types.xml

```diff
@@ -91,6 +92,11 @@
 		<autoUpdatePolicy>recommend</autoUpdatePolicy>
 	</channel-type>
 
+	<channel-type id="systemcommand">
+		<label>Send system command</label>
+		<description>Sends a system command to Kodi. This allows to shutdown/suspend/hibernate/reboot/quit Kodi.</description>
```

Comment:
I really don't care, but I don't think there is some interaction with code. Is there? :wink:

Reply:
No, just copy & paste from the binding where the issue has its origin.